health-services/project-factory/src/server/processFlowClasses/processFlowClassChecker.ts

```
@@ -0,0 +1,48 @@
+
+    // Collect diagnostics
+    const diagnostics = ts.getPreEmitDiagnostics(program);
+
+    if (diagnostics.length > 0) {
+        const formatted = ts.formatDiagnosticsWithColorAndContext(diagnostics, {
+            getCanonicalFileName: f => f,
+            getCurrentDirectory: () => process.cwd(),
+            getNewLine: () => '\n'
+        });
+        console.error("❌ Type errors in processFlowClasses:\n" + formatted);
+        process.exit(1);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)